backend/fms_core/template_importer/importers/library_qc.py

```diff
@@ -0,0 +1,104 @@
+import copy
+from fms_core.template_importer.row_handlers.library_qc.library_qc import LibraryQCRowHandler
+
+from fms_core.models import PropertyType, Protocol, Process
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.library_preparation import LibraryRowHandler, LibraryBatchRowHandler
+from fms_core.templates import LIBRARY_QC_TEMPLATE
+from collections import defaultdict
+from .._utils import float_to_decimal_and_none, input_to_date_and_none
+
+PROPERTIES_STARTING_INDEX = 5 #TODO verify that this index is correct
+
+# {{TEMPLATE PROPERTY NAME : DB PROPERTY NAME}
+TEMPLATE_PROPERTY_MAPPING = {
+    "Measured Volume (uL)": "Library Measured Volume",
+    "Volume Used (uL)": "Library Volume Used",
+    "Concentration (ng/uL)": "Library Concentration",
+    "Quality Instrument": "Library Quality Instrument",
+    "Quality Flag": "Library Quality QC Flag",
+    "Quantity Instrument": "Library Quantity Instrument",
+    "Quantity Flag": "Library Quantity QC Flag",   
+}
+
+class LibraryQCImporter(GenericImporter):
+    SHEETS_INFO = LIBRARY_QC_TEMPLATE['sheets info']
+
+    def __init__(self):
+        super().__init__()
+        self.properties_starting_index = PROPERTIES_STARTING_INDEX
+        self.initialize_data_for_template()
+
+    def initialize_data_for_template(self):
+        #Get protocol for SampleQC
+        protocol = Protocol.objects.get(name='Library Quality Control')
+
+         #Preload data
+        self.preloaded_data = {'process': None, 'process_properties': {}}
+
+        self.preloaded_data['process'] = Process.objects.create(protocol=protocol,
+                                                                comment='Library Quality Control (imported from template)')
+
+        # Preload PropertyType objects for the sample qc in a dictionary for faster access
+        try:
+            self.preloaded_data['process_properties'] = { property.name: {'property_type_obj': property } for property in
+                                                         list(PropertyType.objects.filter(object_id=protocol.id))}
+        except Exception as e:
+            self.base_errors.append(f'Property Type could not be found. {e}')
+
+
+    def import_template_inner(self):
+        libraries_sheet = self.sheets['LibraryQC']
+        for row_id, row_data in enumerate(libraries_sheet.rows):
+
+            process_measurement_properties = self.preloaded_data['process_properties']
+
+             #Populate process properties
+            for i, (key, val) in enumerate(row_data.items()):
+                if key in TEMPLATE_PROPERTY_MAPPING.keys():
+                    process_measurement_properties[TEMPLATE_PROPERTY_MAPPING[key]]['value'] = val
+
+            sample_container = {
+                'container_barcode': row_data['Library Container Barcode'],
```

Comment:
I've applied str_cast_and_normalize in another PR to row_data values to make sure the input is a string for string inputs. It could be a good idea to do it right away instead of fixing this later.